# this is a message that holds the force at the end effector

std_msgs/Header header
float64 force_x
float64 force_y
float64 force_z